### ISSUE TITLE
Disable console output

### ISSUE THREAD

**Issue by tzipperle** [iiasa/ixmp#84]
Hi,
is there a easy opition to disable all output from the console? 

for e.g.
`... DEBUG at.ac.iiasa.ixmp.database.DbDAO:1695 ...`
`--- Job MESSAGE_run.gms ...`

**Comment by danielhuppmann** [comment]
The Java-logging will be less verbose in the next release, scheduled before the end of the summer. For the GAMS-output, that will be more tricky...

**Comment by gidden** [comment]
There is probably something doable via the GAMS CLI. If so, we could check
the log level and do something intelligent accordingly..

**Comment by tzipperle** [comment]
> The Java-logging will be less verbose in the next release, scheduled before the end of the summer. For the GAMS-output, that will be more tricky...

Is there a planned release date yet?

**Comment by gidden** [comment]
Hi Thomas - we're shooting for the end of the month.

**Comment by tzipperle** [comment]
Thanks @gidden for the information. 

What is the best way to test the new renewable implementation until the release?

`conda uninstall --force message_ix`

`pip install git+https://github.com/iiasa/message_ix.git@hackathon-prs`

Do I have to change also something at the ixmp installation?

**Comment by gidden** [comment]
You will need the latest ixmp `master` branch, but I think that is all. @danielhuppmann the ixmp master branch is aligned with `hackathon-prs` right?

**Comment by gidden** [comment]
Ah, also if you want to test out disabling console output, you will need iiasa/ixmp#88

**Comment by gidden** [comment]
which you can do with `ixmp.Platform.set_log_level('NOTSET')`

**Comment by tzipperle** [comment]
Installation workflow:
```
conda uninstall --force ixmp
conda uninstall --force message-ix

```
install ixmp
```
git clone https://github.com/danielhuppmann/ixmp.git
git checkout newjar
cd /ixmp && python setup.py install
```

install message_ix
```
git clone https://github.com/iiasa/message_ix.git /message_ix
git checkout hackathon-prs
cd /message_ix && python setup.py install
```

Error:

> --- MESSAGE_run.gms(199) 4 Mb
>     ***
>     *** Abort "The MESSAGEix version and the MESSAGE-scheme definition in the installed ixmp package do not match!"
>     ***

I also have to remove my local database due to migration issues.

**Comment by danielhuppmann** [comment]
yes, #88 brings in a extended MESSAGE-scheme (v1.1) - you'll need the branch from https://github.com/iiasa/message_ix/pull/113 to be in sync between MESSAGEix and ixmp.

Can you send me a screenshot of the flyway migration issues with your local database?

**Comment by gidden** [comment]
@danielhuppmann I guess we will see this when we rerun CI?

**Comment by danielhuppmann** [comment]
The problem with the checksum mismatch is due to different line endings (LF vs. CRLF) - before, that was local-system-dependent, but it will be consistent going forward. Otherwise, it would not be possible to hand over hsqldb files between Windows and Linux.

Two options for now: if you have nothing important in your local db, delete and start fresh. Otherwise, open the HSQLDB with your favourite sql dev tool and update the `schema_version` table manually.

**Comment by tzipperle** [comment]
I have tried the logger option. But the name `NOTSET` is maybe a little bit confusing. In python `NOTSET` allows all levels.

```
import logging

logging.basicConfig(level=logging.NOTSET)
log = logging.getLogger()

log.debug('debug')
log.info('info')
log.warning('warning')
log.error('error')
log.critical('critical')
```
